### PR TITLE
Navbar

### PR DESCRIPTION
**Pull request by xleonx0x**
added navbar items

### CODE REVIEW

File: frontend/src/components/Navbar/Navbar.tsx
Comment (lachlanshoesmith) [inline frontend/src/components/Navbar/Navbar.tsx:1]: indentation is a little inconsistent, ensure you run this through prettier

**Review by lachlanshoesmith — CHANGES_REQUESTED** [review]

File: frontend/src/components/Navbar/Navbar.tsx
Comment (xleonx0x) [reply frontend/src/components/Navbar/Navbar.tsx:1]: I'm not too familiar with prettier, could you walk me through 😔

Comment (lachlanshoesmith) [reply frontend/src/components/Navbar/Navbar.tsx:1]: 1. merge in latest changes from main to this branch, which will update both package.jsons to include some prettier config
2. install prettier extension in vscode
3. in vscode config, set `editor: default formatter` to prettier
4. enable `editor: format on save`
5. save the troublesome file and the formatting should fix itself

Comment (xleonx0x) [reply frontend/src/components/Navbar/Navbar.tsx:1]: > merge in latest changes from main to this branch, which will update both package.jsons to include some prettier config

how do i exactly do this, i don't wanna mess up anything

Comment (lachlanshoesmith) [reply frontend/src/components/Navbar/Navbar.tsx:1]: `git merge main` should do it

Comment (xleonx0x) [reply frontend/src/components/Navbar/Navbar.tsx:1]: done 👍

**Review by lachlanshoesmith — APPROVED** [review]

**Pull request merged**